react 17.0

Hello, I want to try the library but npm cannot resolve the dependency for react, can you update to the latest deps? Thanks!

I got this when running npm install

```npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.8.0" from use-simple-infinite-scroll@1.0.1
npm ERR! node_modules/use-simple-infinite-scroll
npm ERR!   use-simple-infinite-scroll@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.```

--- a/package.json
+++ b/package.json
@@ -88,7 +88,7 @@
     "typescript": "^3.9.7"
   },
   "peerDependencies": {
-    "react": "^16.8.0"
+    "react": ">=16.8.0 <=17.x"
   },
   "husky": {
     "hooks": {

